zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -100,6 +101,35 @@ void beforeEach() {
         .thenReturn(CONFIG_PREFIX + "-template_index_qualified_name");
   }
 
+  @TestTemplate
+  void shouldOpenDifferentPartitions(
+      final ExporterConfiguration config, final SearchClientAdapter ignored) {
+    // given
+    final var p1Exporter = new CamundaExporter();
+    final var p1Context = getContextFromConfig(config, 1);
+    p1Exporter.configure(p1Context);
+
+    final var p2Exporter = new CamundaExporter();
+    final var p2Context = getContextFromConfig(config, 2);
+    p2Exporter.configure(p2Context);
+
+    // when
+    new Thread(
+            () -> {
+              final var p1ExporterController = new ExporterTestController();
+              p1Exporter.open(p1ExporterController);
+            })
+        .start();
```

Comment:
:upside_down_face: 
```suggestion
    CompletableFuture.runAsync(() -> p1Exporter.open(new ExporterTestController()));
```

Reply:
:question: If you don't join on the thread (or future with the suggestion), how do you guarantee that this runs before the code below, or even that you wait for the P1 exporter to be opened? I don't understand what we want to this in this case.

Reply:
The idea is to run in between and produce a more or less failing test

Reply:
The issue only happens on concurrent run

Reply:
We're still not guaranteeing they run at the same time, but I'm also not sure how we can do this without more granular jobs I guess...